### ISSUE TITLE
Wrong containerd version in release notes for release v17.03.1-ce

### ISSUE THREAD

**Issue by z00sts** [moby/moby#32297]
Wrong containerd version in release notes for release v17.03.1-ce:
```
Update containerd to 595e75c212d19a81d2b808a518fe1afc1391dad5 #31662
```
(https://github.com/docker/docker/pull/31662/commits/74c52a7bab45f63defdbe4c33ec5c77416f2ecf7)

But here version changed to 4ab9917febca54791c5f071a9d1f404867857fcc:
https://github.com/docker/docker/commit/00132cc4426d7914822b98f952d32a45b015485e

**Comment by thaJeztah** [comment]
Thanks! I'll open a PR to fix it, and will edit the release notes on GitHub 👍

**Comment by thaJeztah** [comment]
Release notes are updated; pull request is https://github.com/docker/docker/issues/32298 to fix it in the 17.03.x branch
